perma_web/perma/views/user_management.py

```diff
@@ -1484,6 +1484,12 @@ def email_registrar_request(request, pending_registrar):
     Send email to Perma.cc admins when a library requests an account
     """
     host = request.get_host()
+    try:
+        email = request.user.email
+    except AttributeError:
+        # User did not have an account
+        email = request.POST.get('a-email')
+
     send_admin_email(
         "Perma.cc new library registrar account request",
         pending_registrar.email,
```

Comment:
I think if you switch this to "email" from pending_registrar.email, people will get all the benefit of the nice text you added to the email, AND the reply-to will be the user's email address.